core/bn.js

```diff
@@ -88,12 +88,15 @@ sjcl.bn.prototype = {
   /**
    * Convert to a hex string.
    */
-  toString: function() {
-    this.fullReduce();
-    var out="", i, s, l = this.limbs;
-    for (i=0; i < this.limbs.length; i++) {
+  toString: function(doTrim) {
+    var i, s, l, out = new sjcl.bn(this);
+    out.fullReduce();
+	doTrim && out.trim();
+    l = out.limbs;
+	out = "";
```

Comment:
This also changes the type of the `out`-Variable which makes all major browsers deoptimize the function.